src/modules/userSchemas.ts

```diff
@@ -48,6 +48,7 @@ export const getUserListSchema = z.object({
     active: z.boolean().optional(),
     cursor: z.string().nullish(),
     mailingSettings: z.enum(mailingSettingType).optional(),
```

Comment:
shoud be `mailingSettings: {type: ..., orgId: ...}.optional()` because they only used together